ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -95,6 +95,30 @@
   with_items: "{{ invokerInfo }}"
   when: not invoker.allowMultipleInstances and item.Names[0] != "/invoker{{ groups['invokers'].index(inventory_hostname) }}"
 
+- name: "determine 'docker' binary location"
+  shell: which docker
+  register: dockerBinary
+
+- name: "determine 'docker-runc' binary location"
+  shell: which docker-runc
+  register: runcBinary
+  failed_when: false
+
+- set_fact:
+    runcMount: "-v {{ runcBinary.stdout }}:/usr/bin/docker-runc"
+  when: runcBinary.rc == 0
+
+- name: copy runc script to remote
+  copy:
+    src: "files/docker-runc"
+    dest: "{{ nginx.confdir }}"
```

Comment:
nginx?

Reply:
Just for quick prototyping, I needed a folder which was already there and setup.